README.md

```diff
@@ -264,6 +266,99 @@ It is possible that a standby trying to activate is not able to immediately acqu
 it goes back into standby mode, but if it fails to receive heartbeats from an active server, it will try again to
 acquire the store lock. The interval is random but as of now set to a bit more than a second.
 
+## Static Channels aka Partitioning
```

Comment:
What about "Partitioning" then? I would also change the `static_channels` parameter to the boolean `partitioning`?